### PR TITLE
Update H1 load-tests to the latest changes

### PR DESCRIPTION
**Pull request by ayeshLK**
Related to [#3199](https://github.com/ballerina-platform/ballerina-standard-library/issues/3199)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~

### CODE REVIEW

File: load-tests/h1_h1_passthrough/src/h1_h1_passthrough.bal [hunk 1]
```diff
@@ -17,36 +17,30 @@
...
 
-listener http:Listener securedEP = new(9090, {
-    secureSocket: {
+listener http:Listener securedEP = new (9090,
+    httpVersion = "1.1",
+    secureSocket = {
         key: {
             path: "./security/ballerinaKeystore.p12",
-            password: "ballerina"
+        password: "ballerina"
```
Comment (chamil321) [inline load-tests/h1_h1_passthrough/src/h1_h1_passthrough.bal:25]:
```suggestion
            password: "ballerina"
```

File: load-tests/h1_h1_passthrough/src/h1_h1_passthrough.bal [hunk 2]
```diff
@@ -17,36 +17,30 @@
...
-        } else {
-            log:printError("Error at h1_h1_passthrough", 'error = response);
-            http:Response res = new;
-            res.statusCode = 500;
-            res.setPayload(response.message());
-            error? result = caller->respond(res);
-        }
+    resource function post .(http:Caller caller, http:Request clientRequest) returns error? {
+        http:Response response = check nettyEP->forward("/service/EchoService", clientRequest);
+        return caller->respond(response);
```
Comment (chamil321) [inline load-tests/h1_h1_passthrough/src/h1_h1_passthrough.bal:44]: Let's return the response rather using caller

File: load-tests/h1_transformation/src/h1_transformation.bal [hunk 3]
```diff
@@ -18,56 +18,58 @@ import ballerina/http;
...
             xml|xmldata:Error? xmlPayload = xmldata:fromJson(payload);
-            if (xmlPayload is xml) {
+            if xmlPayload is xml {
                 http:Request clinetreq = new;
                 clinetreq.setXmlPayload(xmlPayload);
                 http:Response|http:ClientError response = nettyEP->post("/service/EchoService", clinetreq);
-                if (response is http:Response) {
-                    error? result = caller->respond(response);
+                if response is http:Response {
+                    return caller->respond(response);
```
Comment (chamil321) [inline load-tests/h1_transformation/src/h1_transformation.bal:52]: Here also, simply return http:Response

File: load-tests/h1c_h1c_passthrough/src/h1c_h1c_passthrough.bal [hunk 4]
```diff
@@ -15,21 +15,12 @@
...
-            log:printError("Error at h1c_h1c_passthrough", 'error = response);
-            http:Response res = new;
-            res.statusCode = 500;
-            res.setPayload(response.message());
-            error? result = caller->respond(res);
-        }
+service /passthrough on new http:Listener(9090, httpVersion = "1.1") {
+    resource function post .(http:Caller caller, http:Request clientRequest) returns error? {
+        http:Response response = check nettyEP->forward("/service/EchoService", clientRequest);
+        return caller->respond(response);
```
Comment (chamil321) [inline load-tests/h1c_h1c_passthrough/src/h1c_h1c_passthrough.bal:24]: Here also, simply return http:Response

File: load-tests/h1c_transformation/src/h1c_transformation.bal [hunk 5]
```diff
@@ -18,39 +18,39 @@ import ballerina/http;
...
-                error? result = caller->respond(res);
+                return caller->respond(res);
             }
         } else {
             log:printError("Error at h1c_transformation", 'error = payload);
             http:Response res = new;
             res.statusCode = 400;
             res.setPayload(payload.message());
-            error? result = caller->respond(res);
+            return caller->respond(res);
```
Comment (chamil321) [inline load-tests/h1c_transformation/src/h1c_transformation.bal:53]: Here also, simply return http:Response

File: load-tests/h1_h1_passthrough/src/h1_h1_passthrough.bal (hunk 1 above)
Comment (ayeshLK) [reply load-tests/h1_h1_passthrough/src/h1_h1_passthrough.bal]: Fixed with [03ab7a3739f257aa4ac62770af56a7b3eff4804e](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/03ab7a3739f257aa4ac62770af56a7b3eff4804e)

File: load-tests/h1_h1_passthrough/src/h1_h1_passthrough.bal (hunk 2 above)
Comment (ayeshLK) [reply load-tests/h1_h1_passthrough/src/h1_h1_passthrough.bal]: Fixed with [03ab7a3739f257aa4ac62770af56a7b3eff4804e](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/03ab7a3739f257aa4ac62770af56a7b3eff4804e)

File: load-tests/h1c_h1c_passthrough/src/h1c_h1c_passthrough.bal (hunk 4 above)
Comment (ayeshLK) [reply load-tests/h1c_h1c_passthrough/src/h1c_h1c_passthrough.bal]: Fixed with [03ab7a3739f257aa4ac62770af56a7b3eff4804e](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/03ab7a3739f257aa4ac62770af56a7b3eff4804e)

File: load-tests/h1c_transformation/src/h1c_transformation.bal (hunk 5 above)
Comment (ayeshLK) [reply load-tests/h1c_transformation/src/h1c_transformation.bal]: Fixed with [03ab7a3739f257aa4ac62770af56a7b3eff4804e](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/03ab7a3739f257aa4ac62770af56a7b3eff4804e) and [3aeae3bb7dcddc0e29f8675c34b21ba3ea3e9684](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/3aeae3bb7dcddc0e29f8675c34b21ba3ea3e9684)

File: load-tests/h1_transformation/src/h1_transformation.bal (hunk 3 above)
Comment (ayeshLK) [reply load-tests/h1_transformation/src/h1_transformation.bal]: Fixed with [03ab7a3739f257aa4ac62770af56a7b3eff4804e](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/03ab7a3739f257aa4ac62770af56a7b3eff4804e) and [3aeae3bb7dcddc0e29f8675c34b21ba3ea3e9684](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/3aeae3bb7dcddc0e29f8675c34b21ba3ea3e9684)

File: load-tests/h1_transformation/src/h1_transformation.bal [hunk 6]
```diff
@@ -18,56 +18,58 @@ import ballerina/http;
 import ballerina/log;
```
Comment (chamil321) [inline load-tests/h1_transformation/src/h1_transformation.bal:18]: Shall we remove log usage from load tests?

**Review by chamil321 — APPROVED** [review]

File: load-tests/h1_transformation/src/h1_transformation.bal (hunk 6 above)
Comment (ayeshLK) [reply load-tests/h1_transformation/src/h1_transformation.bal]: Fixed with [f771696eb0eff66f228fdeee068f227d6ac91663](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/f771696eb0eff66f228fdeee068f227d6ac91663)

**Pull request merged**